code/game/objects/items/storage/pouch.dm

```diff
@@ -114,31 +114,28 @@
 	new /obj/item/tool/weldingtool(src)
 
 
-
-
 /obj/item/storage/pouch/firstaid
 	name = "first-aid pouch"
-	desc = "Standard marine first-aid pouch. It can contain autoinjectors, ointments, and bandages."
+	desc = "Standard marine first-aid pouch. It can contain autoinjectors, sets of pills, and bandages."
 	icon_state = "firstaid"
-	storage_slots = 5
+	storage_slots = 6
 	can_hold = list(
-		/obj/item/stack/medical/heal_pack/ointment,
 		/obj/item/reagent_containers/hypospray/autoinjector,
-		/obj/item/stack/medical/heal_pack/gauze,
-		/obj/item/storage/pill_bottle/packet/tricordrazine,
-		/obj/item/stack/medical/splint,
+		/obj/item/stack/medical,
+		/obj/item/storage/pill_bottle,
```

Comment:
this makes the first aid pouch absolutly OP, especially if you are a early joiner